locales/en/main.ftl

```diff
@@ -67,4 +67,37 @@ admins = Admins
 loading = Loading
 home-feed = Home Feed
 upvote = Upvote
-downvote = Downvote
\ No newline at end of file
+downvote = Downvote
+local-subscribers = Local Subscribers
+subscribers = Subscribers
+community-stats = Community Stats
+moderators = Moderators
+now = Now
+years-ago = {$years ->
+    [one] {$years} year ago
+    *[other] {$years} years ago
+}
```

Comment:
This style of singular or plural is where fluent is starting to show its edge over i18n. After looking at the fluent syntax guide, I discovered that [fluent supports number formatting that uses a similar API to browsers' `Intl.NumberFormat` class](https://projectfluent.org/fluent/guide/functions.html#number). It may even be possible to remove the dependency on the crate I created because of this, although I haven't tested this yet and will have to see if it behaves as expected.